metadata/release/sample.json

```diff
@@ -42,15 +124,15 @@
             },
             "installer.iso": {
```

Comment:
Should this top level be changed from `installer.iso` to `live.iso`?

Reply:
do we need to change installer-pxe too in that case ?

Reply:
yes we can change installer.iso to live.iso